bbj-vscode/test/validation.test.ts

```diff
@@ -395,4 +395,18 @@ describe('BBj validation', async () => {
             node: symbolicLabelRef
         });
     });
+
+    test('Issue 207 about access level from method that overrides auto-getter or setter', async () => {
+        const validationResult = await validate(`
+        class public Issue
+            field protected String Test
+            method public String getTest()
+                methodret #Test
+            methodend
+        classend
+        t! = new Issue()
+        ? t!.getTest()
+        `);
+        expect(validationResult.diagnostics.map(m => m.message).every(m => m === "Could not resolve reference to Class named 'String'.")).toBeTruthy();
```

Comment:
Changing `String` type to `Issue` or some other known type would give a better expectation like "expect now issues"

Reply:
Done, thanks.